src/chainstate/burn/db/sortdb.rs

```diff
@@ -858,6 +861,53 @@ const SORTITION_DB_SCHEMA_5: &'static [&'static str] = &[
     );"#,
 ];
 
+// force the node to go and store the memo for stack_stx and delegate_stx
+const SORTITION_DB_SCHEMA_6: &'static [&'static str] = &[
```

Comment:
This looks like a breaking change -- you can't just `DROP TABLE` these two tables on a running node, because block validation depends on them being populated with the latest Bitcoin transaction data.

This branch can't be a point release without a big warning in the CHANGELOG that node operators will need to sync from genesis.  Can you instead just `ALTER TABLE` to add the `memo` column?

Reply:
I can `ALTER TABLE`, but the reason I did `DROP TABLE` is because I thought this change would require a sync from genesis in order to populate the memo fields? 

I am going to assume from your suggestion that it is okay that memo remains unpopulated for already processed transaction.